subprojects/plugins/src/main/java/org/gradle/api/tasks/GroovySourceSet.java

```diff
@@ -24,7 +24,11 @@
 /**
  * A {@code GroovySourceSetConvention} defines the properties and methods added to a {@link SourceSet} by the {@link
```

Comment:
Can you fix this reference too?  I think this is supposed to just be "GroovySourceSet"